### ISSUE TITLE
Default attributes for tokens

### ISSUE THREAD

**Issue by AntiSC2** [Kruptein/PlanarAlly#46]
If you want to track attributes for monsters or other tokens you need to manually add in every attribute for each one.

- [ ] It should be possible to select a template for trackers when editing a token
- [ ] There should already be some default templates that include things like strength, dexterity, hp etc
- [ ] It should be possible to create and save your own templates

**Comment by Kruptein** [comment]
Yes this is something that I would like to provide through a new asset management system as mentioned in #44, but it should also extend to the edit asset dialog itself indeed.

**Comment by W8BSD** [comment]
It would be great if the template values could take dice expressions as well... ie: in the HP tracker, put in "27d20+189" and have it roll the Kraken's HP on drop.

**Comment by Kruptein** [comment]
That's exactly what I had in mind :D

Good to see it's desired by others as well

**Comment by poltergeisha** [comment]
It's killing me that this isn't a thing. All I need is to be able to save which tokens are controlled by each players, and the token size. Having to recreate tokens for my players on each map is the *worst*.

**Comment by Kruptein** [comment]
New features have been on hold for some time now due to two important
architectural changes to the codebase and because I've started working
which means I simply have less time.

I'll try to give this priority
